adagrams/game.py

```diff
@@ -1,11 +1,95 @@
+import random
+import copy
```

Comment:
I'm not seeing where this library is used in the code below, perhaps it was an extraneous import? Always a good idea to read over your code before submitting the PR so you can be sure not to commit unused code!

Reply:
Yes, I tried a different route using deepcopy and mistakenly left that "import copy" in! I'll be sure to proof from the very top to bottom next time!